ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/dps-interfaces.yml

```
@@ -0,0 +1,19 @@
+### DPS interface ###
+
+dps_interfaces:
+  - name: Dps1
+    description: Test DPS Interface
+    shutdown: true
+    ip_address: 192.168.42.42/24
+    flow_tracker:
+      hardware: FT-HW
+      sampled: FT-S
+    # Cannot test all combination of tcp_mss_ceiling with only one Dps interface
+    # available
+    tcp_mss_ceiling:
+      ipv4: 666
+      ipv6: 666
+      direction: ingress
+    # This is a dummy command
+    eos_cli: |
```

To avoid the extra blank line.

```suggestion
    eos_cli: |-
```

---

a542649

---

Doesn't seem like it :)

---

yeah ... the commit is gone hehe .. but this is in the file!

---

toh - wrong file

done there: b9e73067